pkgs/by-name/sd/sdl2-compat/package.nix

```diff
@@ -30,7 +29,6 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [
     cmake
-    ninja
```

Comment:
cmake+ninja is usually way faster than just cmake+make.

Reply:
For some big applications it really makes a difference but regardless I would just leave it here as we plan to use ninja for everything cmake related anyway.